### PR TITLE
docs: how to pause/resume a pipeline

### PR DESCRIPTION
**Pull request by whynowy**
Closes: #333

### CODE REVIEW

**Review by vigith — APPROVED** [review]
should we write when to pause a pipeline v/s delete and recreate? and what is the difference? call out the caveat that when it resumes it will start from the last known state and can read a lot of data from source?

**Comment by vigith** [comment]
also call this out #990 :)

**Comment by whynowy** [comment]
> also call this out #990 :)

We should fix those bugs instead.

**Comment by whynowy** [comment]
> > also call this out #990 :)
> 
> We should fix those bugs instead.

You are right, call out the impact of the WAL.

File: docs/user-guide/reference/configuration/pipeline-operations.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+To pause a pipeline, use the command below, it will bring the pipeline to `Paused` status, and terminate all the running vertex pods.
+
+```bash
+  kubectl patch pl my-pipeline --type=merge --patch '{"spec": {"lifecycle": {"desiredPhase": "Paused"}}}'
+```
+
+Pausing a pipeline will not cause data loss. It does not clean up the unprocessed data in the pipeline, but just terminates the running pods. When the pipeline is resumed, the pods will be restarted and continue processing the unprocessed data.
+
+When pausing a pipeline, it will shutdown the source vertex pods first, and then wait for the other vertices to finish the backlog before terminating them.
```
Comment (whynowy) [inline docs/user-guide/reference/configuration/pipeline-operations.md:25]: There's not a timeout (grace period) for pausing, need to add it.

**Review by vigith — APPROVED** [review]

**Pull request merged**